### DISCUSSION TITLE
Slack OAuth integration

### DISCUSSION THREAD

**Discussion by fergalindez** [strapi/strapi#7528] [Legacy Topics]
Could be possible to generate Slack OAuth integration as you have other providers?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
